sdk2refactor/common/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Don't name files 'refactor'. Refactoring is the process of changing the code; it's not the final result. You should assume these paths are more-or-less here to stay and name them accordingly.

Reply:
Yeaaaahh that directory name is pretty bad.